e2e/docker-compose.yaml

```diff
@@ -40,3 +33,21 @@ services:
     environment:
       POSTGRES_PASSWORD: postgres # pragma: allowlist secret
       POSTGRES_DB: dev
+
+  cypress:
+    image: cypress/included:14.3.0
+    depends_on:
+      - nginx
+    working_dir: /web
+    volumes:
+      - ../web:/web
+    environment:
+      - CYPRESS_baseUrl=http://nginx:80
+      - CYPRESS_VIDEO=false
+    entrypoint: /bin/sh
+    command: >
+      -c "
+      npm ci &&
+      npx cypress run --config-file cypress.config.ts --browser electron &&
+      npx cypress run --config-file cypress.config.ts --browser firefox
```

Comment:
why is dev chrome and release electron/firefox? chrome+firefox for both sounds sensible?

Reply:
Electron is a stripped down version of chrome, it achieves the same thing and it means we don't have to install chrome within the workflow. So it cuts down on action minutes and makes the setup less bloated. Happy to reconsider though if you feel strongly?

Reply:
I think it depends how similar electron and chrome are – if there's almost no difference then very happy to leave as is

Reply:
They both run on chromium and both have full access to the HTML/CSS/JS rendered content. So I'd be inclined to leave this as a simpler setup. If we start to get behaviour differences we can always add chrome on as an extension if needed later.